cockroachcloud/cloud-api.md

```diff
@@ -212,7 +212,7 @@ Where:
 
 ## Get information about a specific cluster
 
-To retrieve detailed information about a specific cluster, make a `GET` request to the `/v1/clusters/{cluster_id}` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](console-access-management.html#service-accounts) to retrieve information about an organization's clusters.
+To retrieve detailed information about a specific cluster, make a `GET` request to the `/v1/clusters/{cluster_id}` endpoint. The service account associated with the secret key must have `ADMIN` or `READ` [permission](managing-access.html#manage-service-accounts) to retrieve information about an organization's clusters.
```

Comment:
In the new auth model, this can be done by Cluster Admin or Cluster Developer roles.